sdk/nuget-content/account-management.ps1

```diff
@@ -1,7 +1,7 @@
 
 function RegisterProfile()
 {
-	$dllpath = "..\lib\net35\AWSSDK.Core.dll"
+	$dllpath = "..\lib\net472\AWSSDK.Core.dll"
 	$sdkassembly = [System.Reflection.Assembly]::LoadFrom($dllpath)
```

Comment:
@normj do you know why this is here? The script doesn't even use $sdkassembly but I figured you added this long ago for some kind of preloading that is needed from AWSSDK.Core.dll for the script to work. Second I am not certain we even need this script anymore.

Reply:
I will do the removal in another PR because now I am thinking that entire `nuget-content` folder should possibly be deleted.